nixos/tests/gitea.nix

```
@@ -1,6 +1,6 @@
 { system ? builtins.currentSystem,
   config ? {},
-  giteaPackage,
+  giteaPackage ? "gitea",
```

I might be misunderstanding what you are trying to do here, but why not `pkgs.gitea`?

```suggestion
  giteaPackage ? pkgs.gitea,
```

Passing a string makes it much more difficult to pass an arbitrary (custom) package </3

---

Generally I don't think it's very nice to pass in a package from a different package-set, but considering that there are a bunch other tests doing the same (and >1 nixpkgs is instantiated anyways), we an save ourselves the discussion and I'll just do it :)